### DISCUSSION TITLE
Harvester transmission manual speed control

### DISCUSSION THREAD

**Discussion by miracleman51** [Mogli12/KeyboardSteer#743] [Previous version]
Hi,

IM having great fun playing with the harvesters.  One think I have noted however is that when Im using the harvest hydro transmission, I can only manually increase or decrease the speed (ratio) when I have selected the header, and have the header attached to the machine.  Its fine when this is in automatic mode however (which I dont use with the harvesters).

Also I find when stationary (neutral) but with the rpm up, the engine load swings up to full load. I find this with the tractors as well.

Thanks

M

**Comment by Mogli12** [comment]
Hi M,

That is the idea of manual mode of IVTs in VCA. The automatic mode never changes the range. 

Best regards

**Comment by miracleman51** [comment]
Thanks for the prompt response!  I think perhaps there is a misunderstanding.  Its not automatic mode there in an issue with.

The issue is that the manual mode of IVT's only work when you have pressed G or whatever and have the implement or header highlighted.  If you dont have a header attached on the harvester or don't have it selected, you can't change the speed.

It works as I would expect if you have a header on and selected however and works like a hydro/IVT should (very well actually).

**Comment by Mogli12** [comment]
Hi,

Did you try to rename your inputBinding.xml? 

Regards, Stefan
